<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover
{
	
	background-color: rgb(208, 208, 208);
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>811</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>380</y>
      <width>51</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QSlider" name="verticalSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>22</width>
       <height>151</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 175);</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>621</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	background-color: rgb(208, 208, 208);
}</string>
     </property>
     <widget class="QPushButton" name="last">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>40</width>
        <height>33</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/last.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="play">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>40</width>
        <height>33</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/icon_play.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="next">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>40</width>
        <height>33</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/next.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>40</width>
        <height>33</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/add_list.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="voice">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>40</width>
        <height>33</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/voice.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="playmodel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>33</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icon/order.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>581</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="nowtime">
        <property name="text">
         <string>00.00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totallabel">
        <property name="text">
         <string>00.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 175);</string>
    </property>
   </widget>
   <widget class="running" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>running</class>
   <extends>QWidget</extends>
   <header location="global">running.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
